examples/step-47/doc/intro.dox

```diff
@@ -349,6 +349,9 @@ where
 and
 @f{align*}{
 \mathcal{F}(v_h)&:=\sum_{K \in \mathbb{T}}\int_{K} v_h f \ dx
+-
+\sum_{e \in \mathbb{F}, e\subset\partial\Omega}
+\int_e \frac{\partial^2 v_h}{\partial \mathbf n^2}} j \ ds
```

Comment:
I guess this term is in the papers but just not our documentation?

Reply:
This term is not in the paper since the paper deals with 0 boundary conditions.
I suppose we have this term in the implementation.

Reply:
Have you thought about whether it's correct?